lib/src/next/v1/nu_router.dart

```diff
@@ -139,18 +140,18 @@ abstract class NuRouter implements INuRouter {
 
   /// Override to true to call and register the routes only after the NuRouter initialization
   bool lazyRouteRegister = false;
-  List<NuRoute> _routes;
-  List<legacy.NuRouter> _legacyRouters;
-  NuvigatorState _nuvigator;
+  List<NuRoute>? _routes;
+  late List<legacy.NuRouter> _legacyRouters;
```

Comment:
Why the late here vs making nullable elsewhere?

Reply:
Late is used when it's not set yet but you can guarantee it's not null when it's used.